src/hooks/useSections.ts

```diff
@@ -9,7 +9,7 @@ import {
   UserDefinedSection,
   Section,
 } from '../types';
-import { getActiveSectionsWithData } from '../utils';
+import { getActiveSectionsWithData, getSearchSuggestionFeatures } from '../utils';
```

Comment:
Should we consider renaming `getSearchSuggestionFeatures`? It also applies to rex now

Reply:
Thoughts on name? Thinking of just "getFeatures"

Reply:
I think that works 👍